addOns/client/client.gradle.kts

```diff
@@ -0,0 +1,37 @@
+version = "0.0.1"
+description = "Exposes client (browser) side information in ZAP"
+
+zapAddOn {
+    addOnName.set("Client Side Integration")
+    zapVersion.set("2.12.0")
```

Comment:
Is there a description element where we should be mentioning that it uses a browser add-on to provide meaningful content?

Reply:
Added above :)